pkg/apiserver/api_installer.go

```diff
@@ -216,18 +216,27 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 	}
 	versionedStatus := indirectArbitraryPointer(versionedStatusPtr)
 	var (
-		getOptions          runtime.Object
-		versionedGetOptions runtime.Object
-		getOptionsKind      string
-		getSubpath          bool
-		getSubpathKey       string
+		getOptions               runtime.Object
+		versionedGetOptions      runtime.Object
+		getOptionsInternalFQKind unversioned.GroupVersionKind
```

Comment:
> What is FQ? Could we expand the acronym?

`FullyQualified`.  It gets a bit long.  @derekwaynecarr preferred not having `GVK`